documentation/internal/Filters-for-Applications.md

```diff
@@ -88,6 +89,20 @@ Options:
 
 - Set of `stage-name` which is deduced per template (application type)
 
+<a name="status"></a>
+
+#### status
+
+Example: `status=draft`
+
+Considered in implementation: **Yes** [#251 - Filters]
+
+Can have combined values: **Yes** e.g: `status=draft,submitted`
+
+Options:
+
+- Set of `status` options
+
```

Comment:
Yeah sure, okay. I think it can't hurt to have a lot of possible filters, even if some of them don't really end up getting used in our "presets". And I think we'll be adding and tweaking them for the next wee while. :)